commands/files/is_hidden.go

```diff
@@ -0,0 +1,19 @@
+// +build !windows
+
+package files
+
+import (
+	"path/filepath"
+	"strings"
+)
+
+func IsHidden(f File) bool {
+
+	fName := filepath.Base(f.FileName())
+
+	if strings.HasPrefix(fName, ".") && len(fName) > 1 {
```

Comment:
this may capture `..` but not `.`. is that intended? (maybe dont want it for either?)